modules/ROOT/pages/_partials/release_notes/sync-gateway-3-2-0-release-note.adoc

```diff
@@ -0,0 +1,215 @@
+== 3.2.0 -- August 2024
+
+[WARNING]
+--
+
+Do not deploy Eventing/Sync Gateway until all SGW nodes are at version 3.2.
+For older Sync Gateway versions that do not write import XATTRs, Eventing functions will encounter infinite recursions and duplicate mutations if deployed in a mixed mode SGW environment.
+This can only happen during a mixed mode of SGW with a 3.2 version and an older version during an upgrade and a new Eventing/Sync Gateway function is deployed.
+
+--
+
+Version 3.2.0 of Sync Gateway delivers the following features and enhancements:
```

Comment:
The main release-defining feature in 3.2.0 (Audit Logging) is sort of buried in a long list of enhancements here.  I think in other releases we've highlighted the larger features at the top of the release notes in a 'New Features' section, possibly with a link to the What's New page?
e.g. https://docs.couchbase.com/sync-gateway/3.0/release-notes.html#lbl-rel300